src/SourceBuild/tarball/content/smoke-test.sh

```diff
@@ -577,6 +577,55 @@ function runXmlDocTests() {
     fi
 }
 
+function runOmniSharpTests() {
+    dotnetCmd=${dotnetDir}/dotnet
+
+    rm -rf workdir
+    mkdir workdir
+    pushd workdir
+
+    curl -sSLO "https://github.com/OmniSharp/omnisharp-roslyn/releases/latest/download/omnisharp-linux-x64.tar.gz"
+
+    mkdir omnisharp
+    pushd omnisharp
+    tar xf "../omnisharp-linux-x64.tar.gz"
+    popd
+
+    for project in blazorserver blazorwasm classlib console mstest mvc nunit web webapp webapi worker xunit ; do
```

Comment:
This list is hardcoded. Any idea on how to compute this dynamically so we don't have to update this every few releases?

Reply:
I have started re-writing these tests with a proper test framework.  This makes it much easier to share this with the project creation tests.  Will cover this with https://github.com/dotnet/source-build/issues/1654